doc/assessment-design.md

```diff
@@ -0,0 +1,126 @@
+# Assessment design
+
+## Example Homework assessment
+
+Below is an example of "Homework 1". The access rules make it always visible by TAs, and it is visible to students for full credit for 11 days (the 18th to 28th inclusive), and then is available to view and practice but for zero credit thereafter. This homework has six questions, each of which can be repeated until `maxPoints` is reached. Because it is `"type": "Homework"`, all students will see all six questions, and they can attempt them without limit. All questions are worth the same number of points because there is no advantage in using a complex point distribution on Homeworks. Students have access to a formula sheet which is the same as the one they will see on the exam, allowing them to get used to it.
+
+```json
+{
+    "uuid": "13c12b31-ca94-492a-bf69-a8f383cbc582",
+    "type": "Homework",
+    "title": "Vector algebra",
+    "set": "Homework",
+    "number": "1",
+    "allowAccess": [
+        {
+            "role": "TA",
+            "credit": 100
+        },
+        {
+            "mode": "Public",
+            "credit": 100,
+            "startDate": "2019-01-18T00:00:01",
+            "endDate": "2019-01-28T23:59:59"
+        },
+        {
+            "mode": "Public",
+            "credit": 0,
+            "startDate": "2019-01-28T23:59:59"
+        }
+    ],
+    "zones": [
+        {
+            "questions": [
+                {"id": "addVectors1",   "points": 1, "maxPoints": 5},
+                {"id": "addVectors2",   "points": 1, "maxPoints": 5},
+                {"id": "dotProduct1",   "points": 1, "maxPoints": 5},
+                {"id": "dotProduct2",   "points": 1, "maxPoints": 5},
+                {"id": "crossProduct1", "points": 1, "maxPoints": 5},
+                {"id": "crossProduct2", "points": 1, "maxPoints": 5}
+            ]
+        }
+    ],
+    "text": "For this homework you can use the <a target=\"_blank\" href=\"<%= clientFilesCourse %>/formulas.pdf\">formula sheet</a>."
+}
+```
+
+## Example Exam assessment
+
+Below is an example of "Quiz 1" on the same topic as "Homework 1" above. The access rules always allow TAs access, and allow students access for full credit in the CBTF with a link to the given `examUuid`.
```

Comment:
Mention that the `examUuid` is obtained by going to the CBTF scheduler. 

https://cbtf.engr.illinois.edu/sched